model/transaction.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const transactionSchema = Schema({
+  startTime: { type: Date, required: true },
+  endTime: { type: Date, required: true },
+  hostId: { type: Schema.Types.ObjectId, required: true },
+  guestId: { type: Schema.Types.ObjectId, required: true },
+  spotId: { type: Schema.Types.ObjectId, required: true },
+  price: { type: Schema.Types.ObjectId, required: true }
```

Comment:
Hey @nharren, wondering about line 12...Does it need an object id for price? Can you clarify for me?